deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Deploy the app in a testing environment.
+# testingId is random to avoid multiple people deploying same version at the same time.
+testingId=$(( RANDOM ))
+versionName="testing-$testingId"
+eval "mvn package appengine:deploy clean -Dapp.deploy.version=$versionName -Dapp.deploy.promote=False"
+
+# Run end-to-end tests.
+failUrls=0
+successUrls=0
+for script in end-to-end-tests/*.sh; do
```

Comment:
Right now there is only one test, right?

Reply:
Right